indico/htdocs/js/indico/modules/registration/form/field.js

```diff
@@ -100,16 +100,7 @@ ndRegForm.controller('FieldCtrl', function($scope, regFormFactory) {
             return 0;
         }
 
-        var extraSlots = item.maxExtraSlots,
-            placesLimit = item.placesLimit;
-
-        if (!placesLimit) {
-            return extraSlots;
-        } else if (placesLimit && (placesLimit < extraSlots)) {
-            return placesLimit;
-        } else {
-           return extraSlots;
-        }
+        return item.maxExtraSlots + 1;
```

Comment:
Function name should be adapted (it's the total number of slots now, not the extra slots)